diff --git a/dist/shared/src/types/props/banner.d.ts b/dist/shared/src/types/props/banner.d.ts
index d2f4dc973a9639b837eae8d79746f841006247d0..b1fdb7d3ac81c8cf98efe56e7f78080874357abc 100644
--- a/dist/shared/src/types/props/banner.d.ts
+++ b/dist/shared/src/types/props/banner.d.ts
@@ -1,4 +1,5 @@
-import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+
+
 import { Cta, SecondaryCta, TickerSettings, Tracking } from './shared';
 import { OphanComponentEvent } from '../ophan';
 import * as z from 'zod';
@@ -102,7 +103,7 @@ export declare const bannerContentSchema: z.ZodObject<{
         type: import("./shared").SecondaryCtaType.ContributionsReminder;
     } | undefined;
 }>;
-export interface BannerProps extends EmotionJSX.IntrinsicAttributes {
+export interface BannerProps extends JSX.IntrinsicAttributes {
     tracking: Tracking;
     bannerChannel: BannerChannel;
     content?: BannerContent;
diff --git a/dist/shared/src/types/props/epic.d.ts b/dist/shared/src/types/props/epic.d.ts
index cbf676de646eed52b884ea891688d6d509bcee82..b2e4698a2567c6041c0a189dfd80d35486f6b407 100644
--- a/dist/shared/src/types/props/epic.d.ts
+++ b/dist/shared/src/types/props/epic.d.ts
@@ -1,4 +1,4 @@
-import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import type { JSX } from '@emotion/react/jsx-runtime';
 import * as z from 'zod';
 import { Stage, Tracking } from './shared';
 import { OphanComponentEvent } from '../ophan';
@@ -7,7 +7,7 @@ export type ArticleCountType = 'for52Weeks' | 'forTargetedWeeks';
 export type ArticleCounts = {
     [type in ArticleCountType]: number;
 };
-export interface EpicProps extends EmotionJSX.IntrinsicAttributes {
+export interface EpicProps extends JSX.IntrinsicAttributes {
     variant: EpicVariant;
     tracking: Tracking;
     countryCode?: string;
diff --git a/dist/shared/src/types/props/header.d.ts b/dist/shared/src/types/props/header.d.ts
index cdb021c85d8b68a867e3bda690cc15174fba7e66..9532cff79854993d0b2e406eb16aa85da2b15ce5 100644
--- a/dist/shared/src/types/props/header.d.ts
+++ b/dist/shared/src/types/props/header.d.ts
@@ -1,4 +1,4 @@
-import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { JSX } from '@emotion/react/jsx-runtime';
 import * as z from 'zod';
 import { OphanComponentEvent } from '../ophan';
 import { Tracking, Cta } from './shared';
@@ -58,7 +58,7 @@ export declare const headerContentSchema: z.ZodObject<{
     } | undefined;
     benefits?: string[] | undefined;
 }>;
-export interface HeaderProps extends EmotionJSX.IntrinsicAttributes {
+export interface HeaderProps extends JSX.IntrinsicAttributes {
     content: HeaderContent;
     tracking: Tracking;
     mobileContent?: HeaderContent;
